cmd/juju/resource/show_service_test.go

```diff
@@ -64,7 +64,7 @@ func (s *ShowServiceSuite) TestInfo(c *gc.C) {
 	c.Check(info, jc.DeepEquals, &jujucmd.Info{
 		Name:    "resources",
 		Aliases: []string{"list-resources"},
-		Args:    "application-or-unit",
+		Args:    "<application or unit>",
```

Comment:
Is that really our standard, instead of application|unit ?
(vs [application|unit] if it is an optional positional argument, etc.)

The only other one I could think of quickly was "juju ssh", but that one uses:
"<[user@]target>"
And then defines <target> in a variety of ways.

Interestingly ssh and bootstrap differ here:
Usage: juju bootstrap [options] [<cloud name>[/region] [<controller name>]]
Usage: juju ssh [options] <[user@]target> [openssh options] [command]

Maybe that should be [<cloud name[/region]> [<controller name>]] 
or 
juju ssh [user@]<target>
?